catkin_tools/jobs/catkin.py

```diff
@@ -102,6 +130,7 @@ def create_catkin_build_job(context, package, package_path, dependencies, force_
         'make',
         [MAKE_EXEC] + make_args,
         cwd=build_space,
+        env=ctr_env if 'test' in make_args else {},
```

Comment:
I see, that's a little confusion since other well known function signatures use `env` as an environment replacement. You might consider renaming this argument to `extra_env` or similar to make it clearer. But this looks ok as is.